### ISSUE TITLE
Edit community a record was deposited under

### ISSUE THREAD

**Issue by stopopol** [EUDAT-B2SHARE/b2share#1441]
Metadata records can now be edited, but the community a record was placed under is still fixed.

**Comment by emanueldima** [comment]
Yes, that's true and it's by design, as changing the community would mean losing the community-specific metadata. Can you describe a specific use case?

**Comment by stopopol** [comment]
Imagine that you uploaded a dataset under the generic eudat domain, but then you realise that was mistake and that it should actually be stored under a different domain, like LTER. In that case you wouldn't even lose community-specific metadata.

**Comment by emanueldima** [comment]
This use case makes sense, yes.
